src/monsters.cpp

```diff
@@ -186,6 +186,121 @@ void MonsterType::createLoot(Container* corpse)
 	corpse->startDecaying();
 }
 
+void MonsterType::clone(MonsterType* root, MonsterType* child, const std::string type) {
+	if (type == "attacks") {
+		if (!child->spellAttackList.empty()) {
+			for (spellBlock_t attacks : child->spellAttackList) {
+				root->spellAttackList.push_back(attacks);
+			}
+		}
+	} else if (type == "defenses") {
+		if (!child->spellDefenseList.empty()) {
+			for (spellBlock_t defenses : child->spellDefenseList) {
+				root->spellDefenseList.push_back(defenses);
```

Comment:
would you mind having a chat on skype with me I'm having some questions regarding that.
skype: evilhero19